pkg/kubelet/apis/cri/v1alpha1/api.proto

```diff
@@ -516,6 +496,12 @@ message LinuxContainerSecurityContext {
     //    (localhost) by name. The possible profile names are detailed at
     //    http://wiki.apparmor.net/index.php/AppArmor_Core_Policy_Reference
     string apparmor_profile = 9;
+    // Seccomp profile for the container, candicate values are:
+    // * runtime/default: the default profile for the container runtime
+    // * unconfined: unconfined profile, ie, no seccomp sandboxing
+    // * localhost/<profile-name>: the profile installed on the node.
+    //   <profile-name> is the full path of the profile.
+    string seccomp_profile = 10;
```

Comment:
Let's name this `seccomp_profile_path`, since we may want to add a structured seccomp profile to the message eventually.